api/roleAccess/dtos/to-role-assignment-dto.ts

```diff
@@ -87,7 +92,9 @@ export function toSARoleAssignmentBody(currentUserId: string, specificAccessData
   };
 }
 
-export function toDenySARoleAssignmentBody(currentUserId: string, allocateRoleData: AllocateRoleData): any {
+export function toDenySARoleAssignmentBody(
+  currentUserId: string, allocateRoleData: AllocateRoleData, extraAttributesForBasicRole: {[x: string]: string | boolean} = {}
+): any {
```

Comment:
Do we allow 'any' datatype?

Reply:
No, we should never use any.
it was there before, so I did not give it attention.
I have modified all the file to not have any "any" type